src/controllers/item.js

```diff
@@ -1,6 +1,14 @@
+const ItemModel = require('../models/item');
+
 module.exports = {
-  getAllItems: (req, res) => {
-    res.send('All items');
+  getAllItems: async (req, res) => {
+    try {
+      const items = await ItemModel.find();
+      if (items.length <= 0) throw new Error('No items found');
+      else res.json(items);
+    } catch (err) {
+      res.status(422).json({ message: err.message });
```

Comment:
You should change this to: `message: err.message ?? err`.

The custom error you are throwing above is just a string, not an object with a message property. But some mongoose internal errors may be objects with message property. So this ensures that the message value is pulled in correctly for the response body.